drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fw/host_rpu_sys_if.h

```diff
@@ -589,10 +632,8 @@ struct nrf_wifi_cmd_sys_init {
 	struct temp_vbat_config temp_vbat_config_params;
 	unsigned char tcp_ip_checksum_offload;
 	unsigned char country_code[NRF_WIFI_COUNTRY_CODE_LEN];
-	unsigned char ant_gain_2g;
-	unsigned char ant_gain_5g_band1;
-	unsigned char ant_gain_5g_band2;
-	unsigned char ant_gain_5g_band3;
+	unsigned char single_band_only;
```

Comment:
This went in 139?

Reply:
yes, this change went in 137. 
In next patch we are changing this to op_band.